includes/bootstrap.php

```diff
@@ -107,3 +108,22 @@ function maybe_delete_data() {
 	// Fire hooks to let plugin know to delete data.
 	do_action( 'safety_net_delete_data' );
 }
+
+/**
+ * Determines if webhooks should be disabled.
+ *
+ * Webhooks will be disabled if we're on staging, development, or local AND they haven't already been disabled.
+ */
+function maybe_disable_webhooks() {
+	// If webhooks have already been disabled, skip.
+	if ( get_option( 'safety_net_webhooks_disabled' ) ) {
+		return;
+	}
+
+	// If we're not on staging, development, or a local environment, return.
+	if ( is_production() ) {
+		return;
+	}
+
+	do_action( 'safety_net_disable_webhooks' );
```

Comment:
> Activate or set to Paused some of the webhooks
> Refresh and confirm they retain their status

I cannot reproduce this feature. I was able to change the status of the Disabled webhook back to Active or Paused.

Reply:
That's the expected behavior :)

The test was to make sure they retain their Active or Paused status and SN doesn't disable them again. Sorry if that wasn't clear.